app/Http/Requests/Settings/SetLoginRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests\Settings;
+
+use App\Auth\Authorization;
+use App\Http\Requests\Session\LoginRequest;
+use Illuminate\Auth\Access\AuthorizationException;
```

Comment:
The base class `BaseApiRequest` throws `UnauthorizedException`. We should use that to stay consistent.

```suggestion
use App\Exceptions\UnauthorizedException;
```

Reply:
Fixed.